supplementary/alica_tracing/src/tracing/Trace.cpp

```diff
@@ -49,6 +49,7 @@ std::string Trace::context() const
 
 void Trace::finish()
 {
+    setTag("end_time", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count());
```

Comment:
Use same method as amr_common

Reply:
Abhishek said on the other PR that he'd prefer that it be milliseconds as an integer, so they're consistent as is